examples/README.md

```diff
@@ -1,5 +1,8 @@
 # Sample message files
 
+> [!IMPORTANT]
+> Any new files added to the `examples/Test/` folder should be updated to have `D` in `MSH-11`. New files added to the `examples/Test/Automated/` folder should be updated to have `N` in `MSH-11`
```

Comment:
looks like the folder got corrected on line 23 but not here on line 4

Reply:
Fixed